### ISSUE TITLE
[Bug Report][3.1.5] Inconsistent v-select focus behaviour in Safari

### ISSUE THREAD

**Issue by Zercerium** [vuetifyjs/vuetify#16739]
### Environment
**Vuetify Version:** 3.1.5
**Vue Version:** 3.2.47
**Browsers:** Safari 16.3
**OS:** Mac OS 10.15.7

### Steps to reproduce
1. Select the v-select (Dropdown Menu)
2. Press KeyUp or KeyDown to get focus on the List
3. Press ESC or ENTER
4. Press Tab

### Expected Behavior
Next element should have the focus ( v-text-field Test2 )

### Actual Behavior
First element has the focus ( v-text-field Test1 )

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqVUstOwzAQ/BXjS0BqYrWAkKq0KqoEfEClHpoeQrIJpn7JdkqrKv+OnYQqfYDEbXdmPTO78uqAn5WKthXgMY4tcMVSC9NEIBRvw1SppmwanlLRdU1vYWfDggLLpwswdhiTE6w3aYBBZtGYOn0zSfAqmKeMFlILmgYDFMwlkzrNpa9fXEnzBn4Fqct2YgG71PhiuZecijJYJxiRo8eVOKOrcTx43MM37YYx6W3uWpNpqiwyYCtPU66ktuiANBSoRoWWHAXuZIEfzqQwFnFToonnb4M3YEyipdQsvwnuvHgr54TwALdaLoSKPo0U7uoHnyXpCJPgMWoQjzkP3yf4w1plxoRUQm3KKJOczBxHdCUs5RDmks/uo1H08ERyamwfj8Dw8F3LLwPaOSZ40BO3tNj/ZtDRTncYPbayHeQlL6SI09+CDjWIHDToP3OfzZ5kP+Mu8nvPOhG1uyWjYmPOzpiZ5oSrn2j/Wsx9ragR8K/X3qQe4Dacs2se4PU3YuUOQw==)

### Other comments
In Edge on the same systems it works as expected.
Probably it's the same error like here for v-combobox https://github.com/vuetifyjs/vuetify/issues/16274

**Comment by Zercerium** [comment]
Probably not nice but a temporary workaround.
Register an event handler on update:menu and call focus on the input element.
It is also possible to write these additional logic into VSelect into the computed property `menu`
```js
const menu = computed({
  get: () => _menu.value,
  set: v => {
    if (_menu.value && !v) {
        let el = vTextFieldRef.value?.$el;
        let input = el?.querySelector("input");
        input?.focus();
    }
    _menu.value = v;
}
```

**Comment by KaelWD** [comment]
Kinda fixed by 9a4caf331d05c31a1223712802babe7eec707d72 and then properly by #17218
